### ISSUE TITLE
ClassCastException

### ISSUE THREAD

**Issue by mrandi** [zalando-stups/stups-spring-oauth2-support#2]
You can reproduce this error by starting up fullstop.

````
2015-06-11 12:42:38.705 ERROR 91325 --- [nio-8080-exec-9] s.s.TokenInfoResourceServerTokenServices : Unable to get 'scope' value from map

java.lang.ClassCastException: java.lang.String cannot be cast to java.lang.Boolean
	at org.zalando.stups.oauth2.spring.server.TokenInfoResourceServerTokenServices.getScopesWithPermissionTrueFromMap(TokenInfoResourceServerTokenServices.java:154)
	at org.zalando.stups.oauth2.spring.server.TokenInfoResourceServerTokenServices.extractAuthentication(TokenInfoResourceServerTokenServices.java:97)
	at org.zalando.stups.oauth2.spring.server.TokenInfoResourceServerTokenServices.loadAuthentication(TokenInfoResourceServerTokenServices.java:88)
	at org.springframework.security.oauth2.provider.authentication.OAuth2AuthenticationManager.authenticate(OAuth2AuthenticationManager.java:83)
...
````

**Comment by lukasniemeier-zalando** [comment]
Isn't this fixed with https://github.com/zalando-stups/stups-spring-oauth2-support/pull/1 ?

**Comment by jbellmann** [comment]
fixed with #1
